website/src/theme/DocPage/Layout/Main/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Main from '@theme-original/DocPage/Layout/Main'
+import Giscus from '@giscus/react'
+import { useColorMode } from '@docusaurus/theme-common'
+
+export default function MainWrapper({ children, ...props }) {
+  const { isDarkTheme } = useColorMode()
+
+  return (
+    <Main {...props}>
+      {children}
+      <div style={{ height: '32px' }} />
+      <div className='row'>
+        <div className='col'>
+          <Giscus
+            repo='software-mansion/protostar'
+            repoId='R_kgDOGw_HxA'
+            categoryId='DIC_kwDOGw_HxM4CBC8m'
+            category='Announcements'
+            mapping='title'
+            reactionsEnabled='0'
+            emitMetadata='0'
+            inputPosition='top'
+            theme={isDarkTheme ? 'dark' : 'light'}
+            lang='en'
```

Comment:
nitpick: you can get this value from Docusaurus: https://docusaurus.io/docs/docusaurus-core#useDocusaurusContext

Reply:
Changed, although this change seems forced to me.